controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -489,6 +481,14 @@ func (r *StorageConsumerReconciler) reconcileConsumerConfigMap(
 				return err
 			}
 		}
+
+		primaryConsumerUID := consumerConfigMap.GetAnnotations()[primaryConsumerUIDAnnotation]
+		primaryRefIndex := slices.IndexFunc(consumerConfigMap.OwnerReferences, func(ref metav1.OwnerReference) bool {
+			return string(ref.UID) == primaryConsumerUID
+		})
```

Comment:
this is almost unreadable, please use our established multiline convention where each arg gets its own line and the closing parentheses get the last line with the same indentation as the function call
```suggestion
		primaryRefIndex := slices.IndexFunc(
			consumerConfigMap.OwnerReferences, 
			func(ref metav1.OwnerReference) bool {
				return string(ref.UID) == primaryConsumerUID
			},
		)
```